specification.md

```diff
@@ -3827,12 +3827,12 @@ Note that if used with `string` type, the invocation of the function is synchron
 If you need to define parameters in your `functionRef` definition, you can define
 it with its `object` type which has the following properties:
 
-| Parameter | Description | Type | Required |
-| --- | --- | --- | --- |
-| refName | Name of the referenced [function](#Function-Definition) | string | yes |
-| arguments | Arguments (inputs) to be passed to the referenced function | object | yes if function type is `graphql`, otherwise no |
+| Parameter    | Description                                                                                                                                  | Type   | Required                                        |
```

Comment:
I am not markdown expert but isnt this type of space formatting harder to maintain than

        | Parameter | Description | Type | Required |
        | --- | --- | --- | --- |

Reply:
There's nothing in the documentation related to this.
I'm using IntelliJ and it complains if the table is not properly formatted. Then it offers an autofix to format the tables. VSCode formatter also format these tables.
The advantage of well formatted tables is that they are much easier to read. But I understand that on IDEs that don't do that automatically it might be harder to maintain.
Since VSCode is a general language IDE, I think we can assume that IDEs in general can easily format tables.

Anyway, I can format all of them or revert it, whatever you guys think is better.

Reply:
IntelliJ has native support to Markdown. https://www.jetbrains.com/help/idea/markdown.html#table
On VSCode I just noticed I have an extension for that: https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter

Reply:
Thank you for the explanation, @hbelmiro! @tsurdilo I'll start working on the CI/build tools to format the docs. Usually, people have a shell script to do the formatting in these repos. I'll look around.

Reply:
i think in end markdown should be human readable so users dont need to see parsed page and still be able to read doc if possible, imo :)

Reply:
yeah I agree. My comment was related to keep the formatting uniform.

Reply:
Can the conversation be marked as resolved, so that we can proceed with merging this PR?

Reply:
i think the tables should be human redable. the formatting seems to make this much harder imo. wdyt?

Reply:
I agree, but with MD they hardly ever are anyways. But yeah, imposing formatting everywhere sounds like an avoidable overhead.

Reply:
@hbelmiro Cheers!!!